src/js/components/Tabs/__tests__/__snapshots__/Tabs-test.js.snap

```diff
@@ -469,54 +469,44 @@ exports[`Tabs Tab 1`] = `
 }
 
 <div
-  className="c0"
+  class="c0"
 >
   <div
-    className="c1 "
+    class="c1 "
     role="tablist"
   >
     <div
-      className="c2 "
+      class="c2 "
     >
       <button
-        aria-expanded={true}
-        aria-selected={true}
-        className="c3"
-        onBlur={[Function]}
-        onClick={[Function]}
-        onFocus={[Function]}
-        onMouseOut={[Function]}
-        onMouseOver={[Function]}
+        aria-expanded="true"
+        aria-selected="true"
+        class="c3"
         role="tab"
         type="button"
       >
         <div
-          className="c4 c5"
+          class="c4 c5"
         >
           <span
-            className="c6"
+            class="c6"
           >
             Tab 1
           </span>
         </div>
       </button>
       <button
-        aria-expanded={false}
-        aria-selected={false}
-        className="c3"
-        onBlur={[Function]}
-        onClick={[Function]}
-        onFocus={[Function]}
-        onMouseOut={[Function]}
-        onMouseOver={[Function]}
+        aria-expanded="false"
+        aria-selected="false"
+        class="c3"
```

Comment:
Seems like this button has missing function props (onBlur, onFocus, onClick, onMouseOut...), is this change could cause a potential regression, or does it have a reasonable explanation?

Reply:
Not a regression. The old test mechanism does snapshots by JSON-izing the component and therefore it lists props with camelCase and a toString of the handler functions. The newer mechanism is basically a more stringified version of the element so you see class vs className etc and the handlers don't show up as properties on the element. It's still the same DOM